ci/datamgr.py

```diff
@@ -416,6 +416,36 @@ def omniscidb(schema, tables, data_directory, **params):
     conn.close()
 
 
+@cli.command()
+@click.option('-h', '--host', default='localhost')
```

Comment:
Can we avoid setting these configs in two places?

Reply:
This is how all other backends implement it. Trying to make this PR focused on initial sql server support.